### PR TITLE
Devex 1205: Get Sassy

### PR DESCRIPTION
**Pull request by jasonfu9966**
[Devex 1205](https://trello.com/c/7dRHyGqV/1205-get-sassy-update-sass-to-sasscompile)

### CODE REVIEW

**Review by pixiwyn — APPROVED** [review]

File: shared/src/business/utilities/generateHTMLTemplateForPDF/generateHTMLTemplateForPDF.ts [hunk 1]
```diff
@@ -7,29 +7,19 @@
...
-          'Error compiling SASS to CSS while generating PDF',
-          err,
-        );
-        return reject(err);
-      }
-      return resolve(result);
-    });
-  });
+  const { css } = sass.compileString(main);
+
```
Comment (TomElliottFlexion) [inline shared/src/business/utilities/generateHTMLTemplateForPDF/generateHTMLTemplateForPDF.ts]: Can we preserve error logging here?

**Review by TomElliottFlexion — APPROVED** [review]

File: shared/src/business/utilities/generateHTMLTemplateForPDF/generateHTMLTemplateForPDF.ts (hunk 1 above)
Comment (jasonfu9966) [reply shared/src/business/utilities/generateHTMLTemplateForPDF/generateHTMLTemplateForPDF.ts]: done!

Comment (TomElliottFlexion) [reply shared/src/business/utilities/generateHTMLTemplateForPDF/generateHTMLTemplateForPDF.ts]: https://media.giphy.com/media/Cgo9XkgzRcTPW/giphy.gif

**Pull request merged**